src/common/snippets/include/snippets/op/perf_count.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace snippets {
+namespace op {
+
+/**
+ * @interface PerfCountBegin
+ * @brief Performance count start time
+ * @ingroup snippets
+ */
+class PerfCountBegin : public ov::op::Op {
+public:
+    OPENVINO_OP("PerfCountBegin", "SnippetsOpset");
+    PerfCountBegin();
+
+    std::shared_ptr<Node> clone_with_new_inputs(const OutputVector& inputs) const override;
+
+    // used for chrono call
+    std::chrono::high_resolution_clock::time_point start_time_stamp = {};
```

Comment:
As discussed, chrono perf count public member is avoided, rdtsc perf count is kept to access directly such to avoid internal function call for light overhead.